package.json

```diff
@@ -43,21 +43,21 @@
   },
   "dependencies": {
     "@babel/eslint-parser": "7.16.5",
-    "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-airbnb": "^19.0.4",
     "eslint-config-prettier": "^7.0.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-inclusive-language": "^2.1.1",
+    "eslint-plugin-inclusive-language": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-markdown": "^2.2.1",
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "eslint-plugin-unicorn": "^36.0.0",
+    "eslint-plugin-unicorn": "^37.0.1",
     "eslint-plugin-you-dont-need-lodash-underscore": "^6.10.0"
   },
   "peerDependencies": {
-    "eslint": "^7.32.0",
-    "eslint-plugin-jest": "^24.1.3",
+    "eslint": "^7.32.0 || ^8.8.0",
```

Comment:
Will `8.x` work as well, or do we specifically need `^8.8.0` ?

Reply:
8.8.0 was out when I created the PR, what is the benefit of widening it?

Reply:
It forces packages to have either v7.32.0 or 8.8.0. If 8.8.1 comes out this repo would need another PR

Reply:
@Matthew-Mallimo with the `^`, that is not an issue